index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+</head>
+<body>
+
+<!--Start navbar-sec-sec-->
+    <nav class="navbar-sec">
+        <div class="welcome">Welcome</div>
+        <div class="icon-menu"> <img src="img/menu-icon.svg" alt=""></div>
+    </nav>
+  <!--End navbar-sec-sec-->
+
+      <!--Start Hero-->
+    <section class="hero-sec">
+        <header>
+            <h1>Hey there. I'm David <br>
+                <span>I'm a software developer</span></h1>
+        </header>
```

Comment:
- [ ] The main title or the intro, shouldn't be wrapped with a header, the header should only be used above as I mentioned in my previous comment, kindly change that. 😁

Reply:
I have made corrections